code/modules/hippie/code/game/objects/bowling.dm

```diff
@@ -0,0 +1,105 @@
+/obj/item/clothing/shoes/bowling
+	icon = 'icons/hippie/obj/sports.dmi'
+	name = "bowling shoes"
+	icon_state = "bowlingshoes"
+	item_color = "bowling"
+	desc = "Made for use in only the finest bowling alleys."
+	permeability_coefficient = 0.01
+	flags = NOSLIP
+	resistance_flags = FIRE_PROOF |  ACID_PROOF
+	permeability_coefficient = 0.01
+	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 0, rad = 0, fire = 100, acid = 100) //No fire or acid damage, this shit's top quality.
+
+/obj/item/clothing/under/bowling
+	name = "bowling jersey"
+	desc = "The latest in kingpin fashion."
+	icon = 'icons/hippie/obj/sports.dmi'
+	icon_state = "bowlinguniform"
+	item_color = "bowling"
+	resistance_flags = FIRE_PROOF |  ACID_PROOF
+	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 0, rad = 0, fire = 100, acid = 100)
+	can_adjust = 0
+	var/canbowl = 1
+
+/obj/item/clothing/under/bowling/proc/canbowl()
+	canbowl = 1
+
+/obj/item/weapon/twohanded/bowling
+	name = "bowling ball"
+	icon = 'icons/hippie/obj/sports.dmi'
+	icon_state = "bowling_ball"
+	desc = "A heavy, round device used to knock pins (or people) down."
+	force_unwielded = 3
+	force_wielded = 6
+	w_class = 3.0
+	throwforce = 0
+	throw_range = 1
+	throw_speed = 1
+	var/prowielded = 0
```

Comment:
Please use `TRUE` or `FALSE`

I would also suggest changing this to `pro_wielded`